netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -710,8 +711,9 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
 
     private final GrpcHttp2ConnectionHandler grpcHandler;
 
-    BufferingHttp2UpgradeHandler(ChannelHandler handler, GrpcHttp2ConnectionHandler grpcHandler) {
-      super(handler);
+    BufferingHttp2UpgradeHandler(ChannelHandler handler, ChannelHandler upgradeHandler,
+                                 GrpcHttp2ConnectionHandler grpcHandler) {
```

Comment:
We typically wrap lines +4 spaces.

Reply:
Note that what is here does adhere to the style guide, but yes, we do tend to wrap with 4 spaces.